worlds/subnautica/__init__.py

```diff
@@ -13,10 +12,8 @@
 from .items import item_table, group_items, items_by_type, ItemType
```

Comment:
Do you not see these `items_by_type` and `ItemType` as unused?
Is your editor not warning you about that?

Reply:
It didn't due to another special rule where imports into init specifically are fine so that other modules can import it from there, but for a world init that doesn't really work well.

Reply:
The common way to deal with that, that ruff and other tools support is to indicate a re-export with `as`
```python
from .items import items_by_type as items_by_type
```